code/modules/mining/mine_items.dm

```diff
@@ -278,153 +278,60 @@
 	icon_state = "capsule"
 	icon = 'icons/obj/mining.dmi'
 	w_class = 1
+	origin_tech = "engineering=3;bluespace=3"
+	var/template_id = "shelter_alpha"
+	var/datum/map_template/shelter/template
 	var/used = FALSE
 
+/obj/item/weapon/survivalcapsule/proc/get_template()
+	if(template)
+		return
+	template = shelter_templates[template_id]
+	if(!template)
+		throw EXCEPTION("Shelter template ([template_id]) not found!")
+		qdel(src)
+
+/obj/item/weapon/survivalcapsule/Destroy()
+	template = null // without this, capsules would be one use. per round.
```

Comment:
It looks like whoever wrote this assumed the `template` var's contents would get deleted. Odd. This line is unnecessary.